shop-the-look/api/config.py

```diff
@@ -0,0 +1,79 @@
+import os
+import base64
+from datetime import datetime, timedelta
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+
+class Settings:
+    def __init__(self):
+        # Google services
+        self.project_id = 'my-multimodal-search-project'
+        self.location = 'us-central1'
```

Comment:
As we discussed, it would be great to swap out all hard-coded values for Environment variables as this is more in line with how the other sample apps work, and closer aligned with[ the 12 factor app pattern](https://12factor.net/).